src/sentry/tasks/post_process.py

```diff
@@ -1515,6 +1514,8 @@ def detect_new_escalation(job: PostProcessJob):
 
 
 def detect_base_urls_for_uptime(job: PostProcessJob):
+    from sentry.uptime.detectors.detector import detect_base_url_for_project
```

Comment:
I've been running into a few import issues, seems simpler to just import inside the function here since no other systems refer to uptime directly